pkgs/development/python-modules/sphinxcontrib-wavedrom/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, setuptools-scm
+, sphinx
+, wavedrom
+, xcffib
+, cairosvg
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-wavedrom";
+  version = "3.0.4";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
We have a new parameter for pyproject.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters